mw-webapp/src/logic/wayPage/WayPage.tsx

```diff
@@ -307,6 +307,7 @@ export const WayPage = (props: WayPageProps) => {
 
                 renderModalContent({
                   description: singleGoalMetric.description,
+                  text: "goal metric",
```

Comment:
How does it work? What if I want to create something with confirm modal?

Reply:
Deleted param text